Updated image utilities and package component. (#28)
Kodular/Documentation: docs/components/utilities/image-utilities.md

# Image Utilities

## Overview
**Image Utilities** is a non-visible component which can load images asynchronous to the specified component. This reduces the loading time of the image making your apps faster.

## Methods

### Load Image Async

![Load Image Async](/assets/images/components/utilities/image-utilities/m_load-image-async.png)

_**\(** Component `component`, Text `path` **\)**_

Loads a new image from the given path to the specified component.

Params               |  []()       
-------------------- | ------- 

> `component`          | **Component:**  The component where the image has to be shown. This is set using the Component type block.(Found a                          the last of all the blocks of a component. )

Reviewer: ```suggestion
`component`          | **Component:**  The desired component where the image has to be shown.
```
